code/script.js

```diff
@@ -0,0 +1,231 @@
+    .then(data => {
+    
+    // create an array with languages used by using the object keys. 
+    const arrayLanguages = (Object.keys(data))
+    arrayLanguages.forEach(item => {
+      document.getElementById(`languages-${reponame}`).innerHTML += `<span class="repo-languages--item">${item}</span>`    
+      console.log('item', arrayLanguages)
+    })    
+  }) 
+}
+
+////////////////////////////////////////////////////////////////////////////
+///////////////////////////      COMMITS DATA     /////////////////////////
+//////////////////////////////////////////////////////////////////////////
+const displayCommits = (commitsUrl, reponame) => {
+  fetch(commitsUrl, options)
+  .then(res => res.json())
+  .then(data => {
+    document.getElementById(`commits-${reponame}`).innerHTML = `
+    <p>
+    <span class="bold-text">Amount of commits:</span> ${data.length}
+    </p>`
+  })
+  .catch(err => console.error(err))
+}
+
+////////////////////////////////////////////////////////////////////////////
+//////////////////////////      COMMENTS DATA     /////////////////////////
+//////////////////////////////////////////////////////////////////////////
+const displayComments = (commentsUrl, reponame) => {
+  fetch(commentsUrl, options)
+  .then(res => res.json())
+  .then(data => {
+    if (data.length > 0) {
+
+      let commentsArray = []
+      
+      data.forEach(comment => {
+        if (comment.user.login !== username) {
+          document.getElementById(`comments-${reponame}`).innerHTML = `
+          <p>
+            <span class="bold-text">Reviewed by:</span> 
+            <a class="nav-link" href="${data[0].user.html_url}"> ${data[0].user.login}</a>
+          </p>
+          `         
+          document.getElementById(`comments-btn-${reponame}`).innerHTML = `
+            <button class="filled-button" id="click-${reponame}">comments 💬</button> 
+          `
+          commentsArray.push(comment.body)
+          document.getElementById(`click-${reponame}`).addEventListener('click', () => {
+            commentsArray.forEach(item => {
+              modalWrapper.style.display = 'block'
+        
+              document.getElementById('modal').innerHTML += `
+              <p><span class="bold-text modal-header">Comment from ${data[0].user.login}</span></p>
+              <p class="modal-text">${item}</p>
+              <br>
+              `
+            })
+           
+          })
+        }
+        
+      })
+    }
+  })
+  .catch(err => console.error(err))
+}
+
+////////////////////////////////////////////////////////////////////////////
+///////////////////////////      NETLIFY LINK     /////////////////////////
+//////////////////////////////////////////////////////////////////////////
+
+const displayLink = (netlifyUrl, reponame) => {
+  
+    document.getElementById(`website-${reponame}`).innerHTML = `
+    <a href="${netlifyUrl}" target="_blank">
+    <button class="outlined-button">visit website</button>
+    </a>
+    `
```

Comment:
This is such a smart manner to save and display the link. I will definitely learn this from you